flux/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@headlamp-k8s/flux",
-  "version": "0.2.0",
```

Comment:
Can you remove the flux commits from this PR? I think this should be just about the image building.